### PR TITLE
Remove searchable text

### PR DESCRIPTION
**Pull request by elioschmutz**

### CODE REVIEW

File: opengever/dossier/tests/test_indexer.py [hunk 1]
```diff
@@ -164,52 +165,27 @@ def test_keywords_field_is_indexed_in_Subject_index(self):
...
-            u'900',
-            index_data_for(self.dossier).get('SearchableText'),
-            )
-
-        self.assertIn(
-            u'9001',
-            index_data_for(self.dossier).get('SearchableText'),
-            )
+        self.assertIn(u'qpr', indexed_value)
+        self.assertIn(u'900', indexed_value)
```
Comment (buchi) [inline opengever/dossier/tests/test_indexer.py:179]: This doesn't make much sense if the following assertion is met. Maybe add whitespaces if you're expecting `'900'` as a word?

File: opengever/dossier/tests/test_indexer.py [hunk 2]
```diff
@@ -292,14 +268,15 @@ def test_returns_filing_number_for_dossiers_with_only_filing_prefix_information(
...
         IFilingNumber(self.dossier).filing_no = self.filing_no
         self.dossier.reindexObject()
+        self.commit_solr()
 
-        searchable_text_data = index_data_for(self.dossier).get('SearchableText')
+        indexed_value = solr_data_for(self.dossier, 'SearchableText')
 
         for segment in self.searchable_filing_no:
-            self.assertIn(segment, searchable_text_data)
+            self.assertIn(segment, indexed_value.lower())
```
Comment (buchi) [inline opengever/dossier/tests/test_indexer.py:282]: Solr's SearchableText should contain the full `searchable_filing_no`. Thus it should be possible to assert containment of the whole filing number.
Isn't `searchable_filing_number` a string? Thus a segment would be a character, which makes this assertion weak.

File: opengever/dossier/tests/test_indexer.py (hunk 1 above)
Comment (elioschmutz) [reply opengever/dossier/tests/test_indexer.py:179]: I'm not sure if I understand your suggestion correctly. I assert the whole fillling_no now...

File: opengever/dossier/tests/test_indexer.py (hunk 2 above)
Comment (elioschmutz) [reply opengever/dossier/tests/test_indexer.py:282]: I'm not sure if I understand your suggestion correctly. I assert the whole external reference now...

File: opengever/dossier/tests/test_indexer.py (hunk 1 above)
Comment (buchi) [reply opengever/dossier/tests/test_indexer.py]: My point was if you do `assertIn(u'9001', indexed_value)` then `assertIn(u'900', indexed_value)` is already covered by the other assertion. There's no way that this assertion can fail while the `9001` assertion succeeds. Asserting the whole external reference is better anyway.

Comment (elioschmutz) [reply opengever/dossier/tests/test_indexer.py]: ah ok. Thanks for the clarification

**Pull request merged**